sound/soc/sof/intel/hda-mlink.c

```diff
@@ -167,6 +167,56 @@ static int hdaml_lnk_enum(struct device *dev, struct hdac_ext2_link *h2link,
 	return 0;
 }
 
+static int check_power_active(u32 __iomem *lctl, int sublink, bool enable)
+{
+	int mask = BIT(sublink) << AZX_ML_LCTL_CPA_SHIFT;
+	int retry;
+	u32 val;
+
+	udelay(3);
```

Comment:
do we need a macro for the 3 us?

Reply:
I need to check what the recommended values are, I don't even recall where this 3 and 150 values come from.

Reply:
done